stoservice/src/main/java/com/tutrit/stoservice/service/CustomerService.java

```diff
@@ -0,0 +1,21 @@
+package com.tutrit.stoservice.service;
+
+import com.tutrit.stoservice.bean.Customer;
+import com.tutrit.stoservice.context.ApplicationContext;
+import com.tutrit.stoservice.repository.CustomerRepository;
+
+import java.util.UUID;
+
+
+import static com.tutrit.stoservice.service.ParseBody.parseBody;
+
+public class CustomerService {
+    public static void saveCustomer(Customer customer) {
+        customer.setId(UUID.randomUUID().toString());
+        customer.setName(parseBody().get("n"));
```

Comment:
I would take the data from what comes into the method, not from a third-party class.